specs/secureboot.table

```diff
@@ -1,7 +1,10 @@
 table_name("secureboot")
 description("Secure Boot UEFI Settings.")
 schema([
-    Column("secure_boot", INTEGER, "Whether secure boot is enabled"),
+    Column("secure_boot", INTEGER, "Whether secure boot is enabled: 0 disabled, 1 full security, 2 medium security (macOS only)"),
```

Comment:
Hrm. This makes for weird compatibility. 

Previously `secure_boot` is a boolean. Now, macOS comes along and we have this enumeration. 

I'm not sure what makes sense. Brainstorming aloud:
* overload `secure_boot` with full security as `1`
* overload `secure_boot` with full security as `2`
* Leave this as boolean, and make a `secure_mode` column, perhaps only for macos?

Reply:
To your first suggestion, I think we shouldn't break anyone's workflow where `secure_boot` as `1` means "full security" because it was a bool or even because people were expecting the integer `1`.

To your second suggestion, while this is more intuitive (`2` is greater than `1` and `full` is greater than `medium`), that's the only benefit I see over the PR's initial schema.

So if not the PR's approach, the next best alternative is probably the added column. Whether we do that or not, I am curious if `medium security` is something from the UEFI standard specification, or if we are trying to assign a value to an Apple-specific configuration option? Because then what do we do when they have Medium-Low Security, or when Windows has a Medium-High Security, etc?

Reply:
Resolving the conversation since we have added the `secure_mode` column to reflect the level.